docs/advanced/witness.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 8
+sidebar_label: Witness Node
+title: "Witness Node"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/advanced/witness/"/>
+</head>
+
+_Available as of v1.3.0_
+
+In edge cases, the witness node is responsible for establishing a member quorum within an etcd cluster and maintaining high availability. An etcd cluster requires a quorum (a majority of nodes) to agree on updates to the cluster state.
+
+Witness nodes do not store any data, but the [hardware recommendations](https://etcd.io/docs/v3.3/op-guide/hardware/) for etcd nodes must still be considered. Using hardware with limited resources significantly affects cluster performance, as described in the article [Slow etcd performance (performance testing and optimization)](https://www.suse.com/support/kb/doc/?id=000020100). 
```

Comment:
```suggestion
A production Harvester cluster has a control plane with three management nodes (or control plane nodes). Each node runs an etcd service to act as a member in Kubernetes' etcd cluster. The node count of the control plane nodes has to be an odd number (it's always 3 in Harvester) to maintain etcd quorum.

However, there are cases in which a user doesn't want a management node to run workloads, or the user wants to create a two-node HA cluster. That's the situation in a Witness node comes into play.

A witness node doesn't run user workload or store any VM data, it's pure purpose is to maintain etcd's quorum. In general, it's OK to use a machine with a lower hardware spec, but the hardware still needs to satisfy etcd's [hardware recommendations](https://etcd.io/docs/v3.3/op-guide/hardware/). A hardware with limited resources significantly affects cluster performance, as described in the article [Slow etcd performance (performance testing and optimization)](https://www.suse.com/support/kb/doc/?id=000020100). 
```

My suggestion is we need to briefly introduce the control plane and then move to witness node role. I add some thoughts, can you see if this works.

Reply:
Thanks for the suggestions. Updated with minor changes.